cmd/skopeo/tags.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/containers/image/v5/docker"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+
+	"io"
+)
+
+// tagListOutput is the output format of (skopeo tags), primarily so that we can format it with a simple json.MarshalIndent.
+type tagListOutput struct {
+	DockerRepository string   `json:",omitempty"`
+	Tags             []string `json:",omitempty"`
```

Comment:
Maybe drop the `omitempty` from annotation from both:
- `DockerRepository` is never going to be empty
- Empty `tags` could happen, but having it explicitly present would be more explicit (and possibly easier to parse to shell script hacks, though enabling such abuse is also somewhat a disadvantage).

Reply:
agree. will update